### PR TITLE
preparing versions for 1.5.0

### PR DESCRIPTION
**Pull request by suvamM**

### CODE REVIEW

File: version.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "1.4.22",
```
Comment (shaopeng-gh) [inline version.json:3]: Was this a bug that it was different ?

Comment (shaopeng-gh) [reply version.json:3]: Is there other "1.4.22" in source code that need to be updated:
image

Comment (suvamM) [reply version.json:3]: @shaopeng-gh yes. we missed updating the version.

Comment (shaopeng-gh) [reply version.json:3]: The test show all passed, was the test not effective, or actually testing something else not related
image

Comment (suvamM) [reply version.json:3]: @shaopeng-gh This test does not seem to execute. But it's a valid concern and I will follow up on what the deal here is.

**Review by shaopeng-gh — APPROVED** [review]
:shipit:

**Pull request merged**